packages/kbn-i18n/GUIDELINE.md

```diff
@@ -283,13 +283,19 @@ Splitting sentences into several keys often inadvertently presumes a grammar, a
 
 ### Unit tests
 
-When testing React component that use the injectI18n higher-order component, use the shallowWithIntl helper function defined in test_utils/enzyme_helpers to render the component. This will shallow render the component with Enzyme and inject the necessary context and props to use the intl mock defined in test_utils/mocks/intl.
+Testing React component that use the injectI18n higher-order component is more complicated because `injectI18n()` creates a wrapper component around the component defined in the app.
+
+Shallow rendering only tests one level deep and we want to test the rendering of the component defined for the app, so we need to access it via the wrapper's `WrappedComponent` property. Its value will be the component we passed into `injectI18n()`.
```

Comment:
__nit:__ `Shallow rendering only tests` - sounds a bit weird, since shallow rendering doesn't _test_ anything, it just .... renders. Could you please rephrase this a bit to make it clear that with shallow rendering only top level component is rendered, that's is a wrapper itself, not the original component?